e2e/cypress/integration/enterprise/guest_accounts/guest_identification_spec.js

```diff
@@ -184,4 +189,74 @@ describe('Guest Accounts', () => {
         });
         cy.apiLogout();
     });
+
+    it('MM-T1411 Update Guest Users in User Management when Guest feature is disabled', () => {
```

Comment:
I understand that this is related to guest accounts but if nothing to do with MFA, can we move it into a separate file so we can easily isolate MFA related test?  Thanks!